p2panda-rs/src/storage_provider/traits/test_utils.rs

```diff
@@ -209,192 +200,79 @@ pub struct TestData {
 /// valid CREATE operation following entries contain duplicate UPDATE operations. If the
 /// with_delete flag is set to true the last entry in all logs contain be a DELETE operation.
 pub async fn populate_test_db(db: &mut TestStore, config: &PopulateDatabaseConfig) {
-    let key_pairs = test_key_pairs(config.no_of_authors);
-
-    for key_pair in &key_pairs {
-        db.test_data
-            .key_pairs
-            .push(KeyPair::from_private_key(key_pair.private_key()).unwrap());
-
-        for _log_id in 0..config.no_of_logs {
-            let mut previous_operation: Option<DocumentViewId> = None;
-
-            for index in 0..config.no_of_entries {
-                // Create an operation based on the current index and whether this document should
-                // contain a DELETE operation
-                let next_operation_fields = match index {
-                    // First operation is CREATE
-                    0 => Some(operation_fields(config.create_operation_fields.clone())),
-                    // Last operation is DELETE if the with_delete flag is set
-                    seq if seq == (config.no_of_entries - 1) && config.with_delete => None,
-                    // All other operations are UPDATE
-                    _ => Some(operation_fields(config.update_operation_fields.clone())),
-                };
-
-                // Publish the operation encoded on an entry to storage.
-                let (entry_encoded, publish_entry_response) = send_to_store(
-                    &db.store,
-                    &operation(
-                        next_operation_fields,
-                        previous_operation,
-                        Some(config.schema.to_owned()),
-                    ),
-                    key_pair,
-                )
-                .await
-                .unwrap();
-
-                // Set the previous_operations based on the backlink
-                previous_operation = publish_entry_response.backlink.map(DocumentViewId::from);
-
-                // If this was the first entry in the document, store the doucment id for later.
-                if index == 0 {
-                    let document_id = Some(entry_encoded.hash().into());
-                    db.test_data.documents.push(document_id.clone().unwrap());
-                }
-            }
-        }
-    }
+    // TODO: Needs reinstating when we deal with https://github.com/p2panda/p2panda/issues/418
+    todo!()
 }
 
 /// Helper method for publishing an operation encoded on an entry to a store.
 pub async fn send_to_store(
```

Comment:
This is the helper methods that needs re-implementing and is blocking us from bringing the tests back.

In the comment I said it's tied to https://github.com/p2panda/p2panda/issues/418 but actually, that is just ideal scenario, we can bring it back without that. But changes made in my follow-up PR will impact here.